tensorrt_llm/_torch/modules/kimi_kda/_kda_kernels.py

```
@@ -0,0 +1,388 @@
+
+import importlib
+from types import ModuleType
+from typing import Optional, Tuple
+
+import torch
+
+from . import _kda_decode
+
+try:
+    from tensorrt_llm._utils import get_sm_version as _tllm_get_sm_version
+except Exception:  # pragma: no cover — source-loader stub path
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)